tips/151023_git-rebase.md

```diff
@@ -18,10 +18,10 @@ Any feedback is highly appreciated.
 4. Create your PR on github
 5. when your PR has been approved, you can squash all your commits  
   * git rebase -i HEAD~3 (3 being the number of commits you want to squash)
-6. now, you can rebase on master
-  * git fetch origin
-  * git rebase origin/master
-  * git push origin my_feature_branch:master
+6. now, you can merge your branch on master
+  * git checkout master
+  * git merge --ff-only my_feature_branch
```

Comment:
It will `Refuse to merge and exit with a non-zero status unless the current HEAD is already up-to-date or the merge can be resolved as a fast-forward`
It means that if you haven't rebased correctly, it will refuse to merge.